pkgs/shells/zsh/zsh-clipboard/default.nix

```diff
@@ -4,13 +4,13 @@ stdenv.mkDerivation rec {
   pname = "zsh-clipboard";
   version = "1.0";
 
-  src = ./.;
+  dontUnpack = true;
 
```

Comment:
Can we put `dontBuild = true;` here?

Reply:
It's already there further down, but `dontUnpack` is also needed.

Reply:
I meant removing the empty line (8th line).

Reply:
I guess that could be done yeah